.zenodo.json

```
@@ -0,0 +1,162 @@
+{
+    "creators": [
+      {
+        "affiliation": "Lawrence Berkeley National Laboratory",
+        "name": "Vay, Jean-Luc"
```

@jlvay what is your ORCiD? :)
```suggestion
        "name": "Vay, Jean-Luc",
       "orcid": "..."
```